### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by sgpinkus**
"Understand the basics" to "Understand Kubernetes". There is no place in the entire docs really to go "Understand the *non* basics". There is one section "Concepts" for better or worse. Don't give the impression there is something else somewhere else. And anyway, this section should aspire to be that cardinal reference. Also change name of weird button "Learn" -> "View" to make it clear this is just a link to a section of the documentation, and be consistent with other weird button links.

### CODE REVIEW

**Comment by celestehorgan** [comment]
Hey @sgpinkus,

I'm sensing that you're pretty frustrated with navigating the documentation, but keep in mind you're asking people who spend many hours per week, often unpaid, to maintain the documentation. The tone you're taking in your issue body is pretty combative and impolite to those people.

I'm going to review this change because I do think you bring up some valid points, but please remember to be kind to the maintainers of this repository and other open source repositories as well.

File: content/en/docs/home/_index.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ overview: >
   Kubernetes is an open source container orchestration engine for automating deployment, scaling, and management of containerized applications. The open source project is hosted by the Cloud Native Computing Foundation (<a href="https://www.cncf.io/about">CNCF</a>).
 cards:
 - name: concepts
-  title: "Understand the basics"
+  title: "Understand Kubernetes"
```
Comment (celestehorgan) [inline content/en/docs/home/_index.md:25]: This is maybe a useful change, but I'll wait for a second opinion.

File: content/en/docs/home/_index.md [hunk 2]
```diff
@@ -22,9 +22,9 @@ overview: >
...
   description: "Learn about Kubernetes and its fundamental concepts."
-  button: "Learn Concepts"
+  button: "View Concepts"
```
Comment (celestehorgan) [inline content/en/docs/home/_index.md:27]: I'd like to see this change removed from the PR, however. "View" implies that everyone can see, when many people use screenreaders and the like. Moreover, I don't think it's a substantive improvement from "Learn".

**Comment by sgpinkus** [comment]
> I'm sensing that you're pretty frustrated with navigating the documentation, but keep in mind you're asking people who spend many hours per week, often unpaid, to maintain the documentation. The tone you're taking in your issue body is pretty combative and impolite to those people.

Sorry if it came across that way, I can be a bit tone deaf sometimes but there was no malice intended. To me good docs are as important as good software, what ever form they take so I appreciate the effort and dedication of document maintainers 100%.

P.S. I think I get frustrated with pretty much all software documentation I read :). Kubernetes docs are pretty good in the scheme of things.

File: content/en/docs/home/_index.md (hunk 2 above)
Comment (sgpinkus) [reply content/en/docs/home/_index.md:27]: There are 8 button links on said page:

- Learn Concepts
- View Tutorials
- Setup Kubernetes
- View Training
- View Tasks
- Contribute to the docs
- Download Kubernetes
- View Reference

All the other "view" buttons except the training one link to a major section of the documentation, as does the "learn" one I updated here. So main motivation was consistency. I think even the vision impaired would appreciate consistent link naming so they can infer said links all link to something they same - a major section of documentation in this case. 

P.S. I'm not a huge fan of the button links as you can probably tell. Just thought this was a minor tweak worth making while at it.

Comment (celestehorgan) [reply content/en/docs/home/_index.md:27]: @sgpinkus Ah, I see, thanks for explaining this. 

I'm going to drop an LGTM on this then, as verbiage is really just a nitpick at this point.

**Pull request merged**